src/main/kotlin/com/wynntils/athena/database/objects/Guild.kt

```diff
@@ -0,0 +1,12 @@
+package com.wynntils.athena.database.objects
+
+import com.wynntils.athena.database.interfaces.DatabaseObject
+
+data class Guild(
+    override val _id: String,
+
```

Comment:
Spaces in the constructor make it seem like the class starts here which really threw me off for a bit haha